### PR TITLE
trlst-34 prevent publish of specific public documents

### PR DESCRIPTION
**Pull request by markhigham**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -38,6 +38,14 @@ build-docker-cli:
 frontend-code-style:
 		docker run -it --rm -v "$(CURDIR):/app" node:stretch-slim sh -c 'cd /app && npm i && npx prettier --check "trade_remedies_caseworker/templates/{static,sass}/**/*.{scss,js}"'
 
+
+flake8:
```
Comment (rossmiller) [inline Makefile:42]: This is fine for now but going forward it's probably worth adding a docker-compose file so you can simplify things.

File: trade_remedies_caseworker/templates/static/javascripts/widgets/ReviewSubmission.js [hunk 2]
```diff
@@ -150,18 +165,38 @@ define(["modules/helpers", "modules/popUps", "modules/logging"], function (
       .setClass("deficient", deficient);
   }
 
+  function parseConfidentialAttribute(target) {
+    const isConfValue = target.closest("tr").attr("data-fileconfidential");
```
Comment (rossmiller) [inline trade_remedies_caseworker/templates/static/javascripts/widgets/ReviewSubmission.js:169]: This would be better as a `let` rather than `const`

**Review by rossmiller — APPROVED** [review]
Very good! An extremely minor tweak and an observation in the comments.

File: Makefile (hunk 1 above)
Comment (markhigham) [reply Makefile:42]: totally my thinking. This is very much a step in that direction

**Pull request merged**